tools/evaluate_tts.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+"""Evaluate different voices for TTS using predefined voices and text snippets.
+
+Read the text snippets from a file called 'tts_voice_lines.txt' in the current directory.
+Read the available voices from the Icespeak package.
+
+The text snippets are read from the file line by line, and each line is used as input
+to the TTS engine. The text snippets are assumed to be in the Icelandic language.
+After generating the audio data for each text snippet, the audio data is saved to a
+file in the current directory. The file name is constructed from the sentence read.
+The wav file is automatically played back after it has been saved.
+
+After the audio has been played back, the user is prompted to enter a rating for the
+voice used to generate the audio. The rating is a number between 1 and 5, where 1 is
+the worst and 5 is the best. The rating is saved to a file called 'tts_ratings.tsv'.
+Each voice is rated for 'naturalness' and 'correctness'.
+
+The user is prompted to enter a rating for each voice for each text snippet.
+The user can also enter 's' to skip the current text snippet, or 'q' to quit the
+evaluation process and 'r' to repeat the current sound.
+
+The results are saved continuously to the file 'tts_ratings.tsv'. If the program is
+interrupted, the results can be read from this file and the evaluation process can
+be resumed at a later time.
+"""
+
+from typing import Any
+
+import sys
+from pathlib import Path
+
+from icespeak import VOICES, TTSOptions, tts_to_file
+from icespeak.cli import play_audio_file
```

Comment:
If this function is being used outside the cli, it should probably find a new home.